azure-kusto-ingest/azure/kusto/ingest/resource_manager.py

```diff
@@ -0,0 +1,99 @@
+"""
+    This module is serve as a cache to all resources needed by the kusto ingest client
+"""
+
+from azure.kusto.data import KustoClient
+from .connection_string import _ConnectionString
+
+from datetime import datetime
+from datetime import timedelta
+import random
+
+class IngestClientResources:
+    def __init__(self,
+                 secured_ready_for_aggregation_queues = None,
+                 failed_ingestions_queues = None,
+                 successful_ingestions_queues = None,
+                 containers = None,
+                 status_table = None
+                 ):
+        self.secured_ready_for_aggregation_queues = secured_ready_for_aggregation_queues
+        self.failed_ingestions_queues = failed_ingestions_queues
+        self.successful_ingestions_queues = successful_ingestions_queues
+        self.containers = containers
+        self.status_table = status_table
+    
+    def is_applicable(self):
+        if self.secured_ready_for_aggregation_queues \
+           and self.failed_ingestions_queues \
+           and self.successful_ingestions_queues \
+           and self.containers \
+           and self.status_table:
+            return True
```

Comment:
indentation is wrong here

Reply:
This can be simplified to:
```python
return (self.secured_ready_for_aggregation_queues, ...)
```

Reply:
I think that what you write is returning a tuple, and i want to return 'True' only if all the resources are valid. 
I changed this check to use the 'all' method.

Reply:
I only included the parentheses so you could still have your multi-line expression and avoid the backslash escapes (IOW I was being sloppy for brevity 😉 ).